projects/Fitlit-part-one.md

```diff
@@ -72,11 +72,12 @@ Webpack is a powerful tool, which you're encouraged to explore more (the Turing
 	- Make sure HTML, JS and CSS files are all in the `/src` directory
 	- You have to `import` any required modules and code for tests into your test files
 
-
 - While developing, run `npm start`. Webpack will create a live version of your site on a local server, where you can see your changes happen in real time. To access it, in your browser, navigate to the `localhost` address that your terminal gives you. Be aware, if you write a breaking change, your server may crash. The terminal will give you some error report about why the crash happened.
+  - The command `control + c` is used to stop the local server.  Just closing the terminal without stopping the server first could mean things continue to run in the background and cause problems. This command is not specific to Webpack; make note of it for future use.  
```

Comment:
Added this note about `control + c` command to stop running a local server.  Also added this to the set up notes in the fitlit-starter-kit repo's README